### ISSUE TITLE
support stereo drumkit for op1 field

### ISSUE THREAD

**Issue by dlarsen5** [schollz/teoperator#15]
new op1 field has stereo 20sec drum kit specs, I've tried on my own to write the aif files but the start/end points aren't correct

**Comment by schollz** [comment]
Would be great to support field but I don't have a field to test with. Happily accepting PR

**Comment by dlarsen5** [comment]
I have a field and am very willing to make a PR but stuck on figuring out start/end points haha I'm not sure quantizing to the nearest 8192 blocks are used for the new version format, I followed your blog post about using xxd on the aif file and checking offsets

so this is to say I'm stuck on the last bit otherwise I believe it's mostly the same format except extending the total sample time from 12 to 20 seconds and having stereo 2 channels

**Comment by schollz** [comment]
Yeah thats the tricky bit! I did a some of trial and error to get it right. Let me know if I can help otherwise

**Comment by dlarsen5** [comment]
I don't want to take up your time but [here](https://abletonregistry.s3.amazonaws.com/tmp/apes+are+us.aif) is one example file of the new stereo field preset kits if you're curious but will continue to try things

**Comment by schollz** [comment]
Cool, thanks! I'll take a look.
